great-idea/index.html

```diff
@@ -13,11 +13,70 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
+    <body>
+      <div class="container">
+        <section class="flex navigation">
```

Comment:
What is the advantage of having two classes here? Did you have to override something in general styles? I would suggest looking at the solution code for something a little more streamlined.

Reply:
I see that you were trying to style display flex once and repeat it's use.